pkgs/development/libraries/hunspell/dictionaries.nix

```diff
@@ -821,4 +823,27 @@ in rec {
     readmeFile = "README_he_IL.txt";
     license = with lib.licenses; [ agpl3Plus ];
   };
+
+  /* THAI */
+
+  th_TH = th-th;
+  th-th = mkDict {
+    pname = "hunspell-dict-th-th";
+    version = "experimental-2021-11-17";
```

Comment:
where does this version come from?

Reply:
Ah, I forgot to update that. The author merged in a typo fix. I updated the hash but not the version.

Reply:
Normally on these types of versions, nixpkgs uses `unstable-${DATE}` but since the project branch is `experimental`, it seemed more appropriate.